projects/dev.yaml

```diff
@@ -37,6 +37,8 @@ paths:
               x-modules:
                 - path: v1/citoid.js
                   options: '{{options.citoid}}'
+                - path: v1/lists.js
+                  options: '{{options.lists}}'
```

Comment:
Just to verify I'm correct - the options are not actually ever used, right? I'm asking cause I don't see them being specified in the `config.wikimedia.yaml` so I guess this was added for the future uses and we don't need to accompany this PR with a prod config change.

Reply:
Yeah, it's not used currently. I will probably use it for [T180416](https://phabricator.wikimedia.org/T180416) but nothing substantial.